YttriumWrapper.podspec

```
@@ -0,0 +1,27 @@
+Pod::Spec.new do |s|
+  s.name         = 'YttriumWrapper'
+  s.version      = '0.0.0'
+  s.summary      = '4337 implementation'
+  s.description  = '4337 implementation '
+  s.homepage     = 'https://reown.com'
+  s.license      = { :type => 'MIT', :file => 'LICENSE' }
+  s.authors      = 'Reown, Inc.'
```

```suggestion
  s.authors      = 'reown inc.'
```

https://reown-inc.slack.com/archives/C04SYT4LU2W/p1727206447394799